sequencing-server/test/command-dictionary.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { GraphQLClient } from 'graphql-request';
+import { insertCommandDictionary } from './testUtils/CommandDictionary.js';
+
+let graphqlClient: GraphQLClient;
+
+beforeEach(async () => {
+  graphqlClient = new GraphQLClient(process.env['MERLIN_GRAPHQL_URL'] as string, {
+    headers: { 'x-hasura-admin-secret': process.env['HASURA_GRAPHQL_ADMIN_SECRET'] as string },
+  });
+});
+
+describe('upload command dictionary', () => {
+  it('should upload a command dictionary and all of the fields should be populated correctly', async () => {
+    expect(await insertCommandDictionary(graphqlClient)).toBeDefined();
+  }, 30000);
```

Comment:
Could you please explain how the "and all of the fields should be populated correctly" part of the test name is asserted?

Reply:
I missed the rest of the test changes, it does a strict equality check from the `parsed_json` column to parsing our sample command dictionary on the fly.